### ISSUE TITLE
Error: build failed: rpc error: code = Internal desc = cannot create build volume: cannot create build volume: %!w(string=)

### ISSUE THREAD

**Issue by ronys** [gitpod-io/gitpod#1545]
### Describe the bug
Creating a workspace failed with above message

### Steps to reproduce
1. Create a workspace via https://gitpod.io/#https://github.com/pwsafe/pwsafe/tree/gitpod2
The .gitpod.Dockerfile file has a bug that causes the build process to get stuck waiting for user input.
2. Fix commit and push.
3. Try to create workspace using same url as in 1: result as in title.
### Expected behavior
Either a workspace should be successfully created or a clear error message describing what happened and why (and ideally how to fix this) should be displayed.

### Additional information
See https://community.gitpod.io/t/workspace-image-creation-stuck-waiting-for-input/1381/2

### Example repository
https://github.com/pwsafe/pwsafe/tree/gitpod2

**Comment by Kreyren** [comment]
Unable to reproduce on example repository:

![image](https://user-images.githubusercontent.com/11302521/82843980-27273d00-9edf-11ea-8ad3-35f8770bcd37.png)

@ronys Elaborate on reproduction of this issue

**Comment by ronys** [comment]
This happened when trying to create a workspace with a "broken" Dockerfile that caused the creation process to get stuck, followed by an attempt to create a workspace from the same branch after a commit that fixed the problem in the Dockerfile.

After a few hours, the problematic workspace timed out with a "failed" status, after which I could create a functional workspace with no problem.

If this doesn't point where the problem might be, then there's not much more that I can do to help.

**Comment by Kreyren** [comment]
@ronys Confirm this being a duplicate of https://github.com/gitpod-io/gitpod/issues/1419

I assume that your workspace is running something alike:

```dockerfile
while true do; sleep 30; done
```

Which causes the workspace to get stuck at docker-build phase which makes that workspace unusable?

**Comment by ronys** [comment]
Well, "conceptually" it's the same. My workspace was running debconf as part of installing a required package, and debconf was waiting on stdin until the build timed out.
The fix to the build process was suggested by someone on the forum.
This ticket is more about the failure mode when trying to create a new workspace while another is stuck.

**Comment by Kreyren** [comment]
What do you propose then?

Alternatively you can use
```dockerfile
# To avoid bricked workspaces (https://github.com/gitpod-io/gitpod/issues/1171)
ENV DEBIAN_FRONTEND=noninteractive
```

which works as a permanent fix on debian, but may not be a solution on ubuntu.

(nothing is a solution on ubuntu btw. it's a little known fact to everyone who used it)

**Comment by ronys** [comment]
Well, my issue is resolved, but I'd look at the workspace generation flow where a user retries to create a "stuck" workspace. At the very least, a more user-friendly error message than the subject of this ticket could be displayed.

(Share your opinion re Ubuntu, but vox popoli etc.)

**Comment by Kreyren** [comment]
> Well, my issue is resolved, but I'd look at the workspace generation flow where a user retries to create a "stuck" workspace. @ronys 

Relevant to https://github.com/gitpod-io/gitpod/issues/1419

> At the very least, a more user-friendly error message than the subject of this ticket could be displayed. @ronys 

Follows up on https://github.com/gitpod-io/gitpod/issues/1556

---

Closing this as solved

**Comment by jankeromnes** [comment]
It seems that a server in the EU ran out of disk space and slipped under the radar. We've resolved the situation, so this issue should be fixed again.

See also: https://github.com/gitpod-io/gitpod/issues/1559#issuecomment-635239763

**Comment by Kreyren** [comment]
@jankeromnes I believe that this is unrelated to #1559 since end-user reported this issue 3 days ago (the 1559 seems to be a new issue) and it seems to be caused by a different thing.

**Comment by rathermuneer** [comment]
While trying to start a workspace I am facing the same issue as above. Any solutions here?

**Comment by anglinb** [comment]
I'm hitting this too, status site says "Service Under Maintenance" https://www.gitpodstatus.com/

**Comment by svenefftinge** [comment]
Sorry for that! It should be resolved now.
